client/components/thank-you/style.scss

```diff
@@ -5,3 +5,15 @@
 		max-width: 150px;
 	}
 }
+
+.thank-you__container-header {
+	flex-direction: column;
+	text-align: center;
+	align-items: center;
+}
+
+.thank-you__steps-buttons {
+	margin-left: auto;
+	margin-right: 0;
```

Comment:
Question: why do we need `margin-right: 0` here? I am not seeing this overriding any values, so I think it's a no-op.